ceremonies/src/lib.rs

```diff
@@ -89,28 +89,6 @@ pub mod pallet {
 
 	#[pallet::call]
 	impl<T: Config> Pallet<T> {
-		#[pallet::weight(10_000)]
-		pub fn grant_reputation(
-			origin: OriginFor<T>,
-			cid: CommunityIdentifier,
-			reputable: T::AccountId,
-		) -> DispatchResultWithPostInfo {
```

Comment:
I did not see any reason to keep this, do you @brenzi?

Reply:
No